tests/util/test_phaseFunctions.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+
+from astropy import units as u
+from astropy import constants as const
+import EXOSIMS.util.phaseFunctions as pf
+import numpy as np
+
+"""phaseFunctions.py module unit tests
+
+Sonny Rappaport, June 2021 (in format of code for test_deltaMag)
+
+General strategy: I look up the paper referenced in phaseFunctions.py and 
+doublecheck the paper's function vs what phaseFunction method returns. 
+"""
+
+class TestPhaseFunctions(unittest.TestCase):
+    
+    def test1(self):
+        """Testing the quasiLambertPhaseFunction and its inverse for arbitrary small inputs
+          Note: Inverse should only take on input values from 0 to 1.
+          """
+
+        i1 = np.array([0,.1,.2,.3,.4])
+        result = pf.quasiLambertPhaseFunction(i1)
+        expected = np.array([1.,0.9950,0.9802,0.9558,0.9226])
+        np.testing.assert_allclose(expected, result, rtol=1e-1, atol=0)
+
+        i2 = np.array([1.,0.9950,0.9802,0.9558,0.9226])
+        result2 = pf.quasiLambertPhaseFunctionInverse(i2)
+        expected2 = np.array([0,.1,.2,.3,.4]) 
+        np.testing.assert_allclose(expected2, result2, rtol=1e-1, atol=0)
+
+    def test2(self): 
+        """ Testing the hyperbolicTangentPhaseFunc.""" 
```

Comment:
empty functions. you should be able to run the hyperbolic tangent functions the same way as the QL, just with looser errors and using the sample inputs in the phaseFunction script (they should be in there)